pypose/optim/kernel.py

```diff
@@ -255,7 +255,7 @@ def forward(self, input: Tensor) -> Tensor:
         '''
         assert torch.all(input >= 0), 'input has to be non-negative'
         part1 = (1 + ((input-self.a) / self.b).exp()).log()
-        part2 = (1 + (-self.a / self.b).exp())
+        part2 = (1 + torch.tensor(-self.a / self.b).exp()).log()
```

Comment:
the entire item is a scalar, we can pre-compute it in `__init__`  by using the `math` library, as the overhead of small tensor is big.